### PR TITLE
feat(fe): support file transfer in online debug

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
FE of #1217 
related Be pr: https://github.com/apache/apisix-dashboard/pull/1465
___
### New feature or improvement
- Describe the details and related test reports.

suport `Content-type: multipart/form-data` in request expect `GET` and `HEAD`

mmexport1612875222163

___
### Please add the corresponding test cases if necessary.
TODO:
- [ ] add e2e test cases

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Hi, please sync your codes with the master branch to fix the CI failure.

**Comment by liuxiran** [comment]
everything is ready now, please take a look when you are free, thanks

@juzhiyuan @LiteSun @bzp2010

**Comment by juzhiyuan** [comment]
OK

File: web/cypress/integration/route/online-debug.spec.js [hunk 1]
```diff
@@ -40,21 +41,39 @@ context('Online debug', () => {
...
 
+  const domSelector = {
+    debugDraw: '[data-cy=debug-draw]',
+    deubugMethod: '[data-cy=debug-method]',
+    debugProtocol: '[data-cy=debug-protocol]',
+    debugFormDataKey0: '#dynamic_form_data_item_params_0_key',
+    debugFormDataType0: '[data-cy=debug-formdata-type-0]',
+    debugFormDataValue0: '#dynamic_form_data_item_params_0_value',
+    debugFormDataFileButton0: '[data-cy=debug-upload-btn-0]',
+    codeMirrorCOde: '.CodeMirror-code',
```
Comment (LiteSun) [inline web/cypress/integration/route/online-debug.spec.js:59]:
```suggestion
    codeMirrorCode: '.CodeMirror-code',
```

**Review by LiteSun — COMMENTED** [review]
LGTM, Except for the typos.

File: web/cypress/integration/route/online-debug.spec.js (hunk 1 above)
Comment (liuxiran) [reply web/cypress/integration/route/online-debug.spec.js]: thanks, fixed

**Review by LiteSun — APPROVED** [review]

**Comment by liuxiran** [comment]
https://github.com/apache/apisix-dashboard/pull/1466/checks?check_run_id=1984404224
Backend e2e failed, need  @Jaycean to confirm.

**Comment by Jaycean** [comment]
> https://github.com/apache/apisix-dashboard/pull/1466/checks?check_run_id=1984404224
> Backend e2e failed, need @Jaycean to confirm.

OK, let me check the E2E test case

**Comment by membphis** [comment]
Conflicting files, please merge the master branch

**Review by juzhiyuan — APPROVED** [review]

**Comment by liuxiran** [comment]
thanks all reviewers and  @juzhiyuan' s help , pr merged.

**Pull request merged**